spec/poros/directions_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Directions do
+  it 'will get the info necessary from API call', :vcr do
+    address = "1331 17th St LL100, Denver, CO 80202"
+    lat = 39.750284
+    long = -104.995227
+    directions = DirectionsService.get_directions(address, lat, long)
+
+    final = Directions.new(directions)
+
+    expect(final.distance).to eq(0.303)
+    expect(final.time).to eq("00:01:02")
+    expect(final.directions).to eq("Start out going southeast on 17th St toward Larimer St/CO-33.")
+  end
+end
```

Comment:
If for any reason you needed to rerecord this VCR cassette, some of these assertions might no longer pass since time can change for multiple reasons (traffic, weather, etc.). Since you're testing your poro here, I'd hard code a response that the Service could give back, pass it to Directions.new() and then make your assertions.

Reply:
similar to what you're doing in the Station poro spec.